packages/sources/elwood/src/transport/lwba.ts

```diff
@@ -0,0 +1,165 @@
+import { BaseEndpointTypesLwba } from '../endpoint/crypto-lwba'
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports'
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { EndpointContext } from '@chainlink/external-adapter-framework/adapter'
+import axios from 'axios'
+
+const logger = makeLogger('ElwoodWsLwba')
+
+const DEFAULT_TRANSPORT_NAME = 'default_single_transport'
+
+export type SubscribeRequest = {
+  action: 'subscribe' | 'unsubscribe'
+  stream: 'index'
+  symbol: string
+  index_freq: number
+}
+
+export type PriceResponse = {
+  type: 'Index'
+  data: {
+    price: string
+    bid: string
+    ask: string
+    symbol: string
+    timestamp: string
+  }
+  sequence: number
+}
+
+export type HeartbeatResponse = {
+  type: 'heartbeat'
+  data: string
+  sequence: number
+}
+
+export type ErrorResponse = {
+  type: unknown
+  error: Record<string, unknown>
+}
+
+export type ResponseMessage = PriceResponse | HeartbeatResponse
+
+export type WsTransportTypes = BaseEndpointTypesLwba & {
+  Provider: {
+    WsMessage: ResponseMessage
+  }
+}
+
+export const transport: WebSocketTransport<WsTransportTypes> =
```

Comment:
How is this different from a transport used in `crypto` endpoint ? Is it only the top level `result` response that is null in this transport ? Do we want to do this to fix the alert issues with missing/wrong ask/bid/mid prices ? If so can me have the logic in one place and reuse it somehow ?